### ISSUE TITLE
Analytics: Unsupported files appear as unknown and not counted in the Files Processed by Outcome count

### ISSUE THREAD

**Issue by NadHodeGW** [filetrust/program-icap#290]
1-
With the NCFS Policy set to Block for both 'UN-PROCESSABLE FILE TYPES' and 'GLASSWALL BLOCKED FILES'
A non supported file was submitted using the Icap client tool

Expected: 
Viewing the Analytics screen, the count for 'Failed Files Processed by Outcome' is increased by 1

Actual:
Viewing the Analytics screen, the count for 'Failed Files Processed by Outcome' is not updated

2-
With the NCFS Policy set to Relay for both 'UN-PROCESSABLE FILE TYPES' and 'GLASSWALL BLOCKED FILES'
A non supported file was submitted using the Icap client tool

Expected: 
Viewing the Analytics screen, the count for 'Unmodified Files Processed by Outcome' is increased by 1

Actual:
Viewing the Analytics screen, the count for 'Unmodified Files Processed by Outcome' is not updated

![2021-02-15_11h19_24.png](https://images.zenhubusercontent.com/5f315efe0e4f4158677aea6e/93be3bea-e21f-42ed-8561-b9eac595853c)

![2021-02-15_11h19_40.png](https://images.zenhubusercontent.com/5f315efe0e4f4158677aea6e/f932275e-102d-4b93-b478-b511d36fec6c)

**Comment by gw-sbirch** [comment]
@NadHodeGW 

The issue was that Blocked File Action and Umanagemed File Action were both not being used to determin Risk/Outcome, so the fix was to include those in the calculation.

Transaction Log should report them correctly as "Blocked By Policy" or "Allowed By Policy" depending on the NCFS policy.

Also to point out, the two transactions in this issue had the same PolicyId so In this case, they are coming back as Blocked By Policy due to being unknown. This is due to another issue that was occuring at the time of raising. https://github.com/filetrust/icap-management-ui/issues/220

**Comment by asloman** [comment]
With the NCFS Policy set to Block for both 'UN-PROCESSABLE FILE TYPES' and 'GLASSWALL BLOCKED FILES'
A non supported file was submitted using the Icap client tool

Expected:
Viewing the Analytics screen, the count for 'Failed Files Processed by Outcome' is increased by 1

Actual:
Viewing the Analytics screen, the count for 'Failed Files Processed by Outcome' is increased by 1

2-
With the NCFS Policy set to Relay for both 'UN-PROCESSABLE FILE TYPES' and 'GLASSWALL BLOCKED FILES'
A non supported file was submitted using the Icap client tool

Expected:
Viewing the Analytics screen, the count for 'Unmodified Files Processed by Outcome' is increased by 1

Actual:
Viewing the Analytics screen, the count for 'Unmodified Files Processed by Outcome' is increased by 1
